easybuild/easyconfigs/t/texlive/texlive-20200406-GCCcore-8.3.0.eb

```diff
@@ -41,6 +45,7 @@ postinstallcmds = [
     'echo "TEXMFSYSCONFIG %(installdir)s/texmf-config" >> %(installdir)s/texlive.profile && '
     'echo "TEXMFSYSVAR    %(installdir)s/texmf-var" >> %(installdir)s/texlive.profile && '
     '%(builddir)s/install-tl-%(version)s/install-tl -profile %(installdir)s/texlive.profile'
+    ' -repository ' + 'ftp://tug.org/historic/systems/texlive/%s/tlnet-final' % local_version_year
```

Comment:
Is there an HTTPS mirror?

FTP is blocked on some systems (it is on ours)

Reply:
I think there's an https version. I'll change and test tomorrow when I'm back in the office.

Reply:
it http://ftp.math.utah.edu/pub/tex/historic/systems/texlive/ good enough ? There are reasons not to hammer it, but I hope it's ok? https://tex.stackexchange.com/questions/460132/historic-tex-live-distributions-https-sftp-mirror

Reply:
"Maybe rsync://tug.org/historic/ is a better option?" @boegel can you please elaborate? I don't think I understand.

My current thinking is:
```
install-tl ||
install-tl -repository ftp:// ||
install-tl -repository https://
```
given how the https mirror is almost unusably slow.


I would then skip the first line when updating the already old versions.

Reply:
@hattom Makes sense to me. That should lead to a working install even on systems where FTP is blocked.

Reply:
Do use a local variable here you just plug in the URL to use:

```python
local_install_tl = "%%(builddir)s/install-tl-%%(version)s/install-tl -profile %(installdir)s/texlive.profile %s"

postinstallcmds = [
    'echo "TEXDIR         %(installdir)s/" > %(installdir)s/texlive.profile && '
    'echo "TEXMFLOCAL     %(installdir)s/texmf-local" >> %(installdir)s/texlive.profile && '
    'echo "TEXMFSYSCONFIG %(installdir)s/texmf-config" >> %(installdir)s/texlive.profile && '
    'echo "TEXMFSYSVAR    %(installdir)s/texmf-var" >> %(installdir)s/texlive.profile ',
    # sensible comment to explain what the command below is about goes here
    ' || '.join([
        local_install_tl % '',
        local_install_tl % '-repository ftp://...',
        local_install_tl % '-repository https://...',
    ]),
]
```

The use of `%%` in `local_install_tl` is important...